### ISSUE TITLE
Sensitivity to withered crop

### ISSUE THREAD

**Issue by Gonimy-Vetrom** [Mogli12/AIVehicleExtension#227]
When using the "Seasons" mod, crops can dry in small areas (as in the picture). But they do not interfere with the harvester (in the game at least :), but he bypasses them like islands with grass. Is it possible to add a setting or do something else to ignore these areas with a dried crop and go through them? Thank.
![fsScreen_2020_06_28_22_32_51](https://user-images.githubusercontent.com/8200153/86024537-c935ca00-ba35-11ea-88ad-74e5c01edff8.png)
![fsScreen_2020_06_28_22_41_24](https://user-images.githubusercontent.com/8200153/86024735-09954800-ba36-11ea-8215-99f94154cd3b.png)

![fsScreen_2020_06_29_00_15_28](https://user-images.githubusercontent.com/8200153/86024697-fedab300-ba35-11ea-8aa9-9f084988a4ef.png)

**Comment by Randywes** [comment]
Yeah, it happens to me too. and also my main issue is the tractor turns too sharp in rounds radius turns. and what happens instead of continuing the line is Bypassing round and stops. i added a picture below


![Desktop Screenshot 2020 07 07 - 14 02 02 33](https://user-images.githubusercontent.com/54022592/86778817-7b3b4a80-c063-11ea-9fe0-d0b904493e83.png)

**Comment by Gonimy-Vetrom** [comment]
Interestingly, today two combines mowed sunflower, and they coped well with places where the crop withered.
![fsScreen_2020_07_11_18_05_30](https://user-images.githubusercontent.com/8200153/87227267-8adbcb80-c3a2-11ea-94ee-a094400e0a3e.png)
